besu-plugins/linea-sequencer/sequencer/src/main/java/net/consensys/linea/rpc/services/LineaBundleEndpointsPlugin.java

```diff
@@ -43,6 +49,22 @@ public void doRegister(final ServiceManager serviceManager) {
         lineaCancelBundleMethod::execute);
   }
 
+  public PluginTransactionPoolValidator createTransactionValidator() {
+    final var validators =
+        new PluginTransactionPoolValidator[] {
+          new AllowedAddressValidator(transactionPoolValidatorConfiguration().deniedAddresses()),
```

Comment:
@fab-10 in reality how often does this get called (loading the denied addresses) - is it only on startup?